crates/rust-analyzer/src/config.rs

```diff
@@ -141,6 +141,11 @@ config_data! {
         /// their contents.
         highlighting_strings: bool = "true",
 
+        /// Whether to show documentation on hover.
+        hover_documentation: bool   = "true",
+        /// Use markdown syntax for links in hover.
+        hover_linksInHover: bool    = "true",
+
```

Comment:
This PR splits up `hoverActions` from general hover configurations as I feel like a split makes more sense here. This does move ` hoverActions_linksInHover` to ` hover_linksInHover` however which is a breaking change. I suppose we can keep both settings and mark the old version deprecated?

Reply:
Implemented configuration deprecation now https://github.com/rust-analyzer/rust-analyzer/pull/9264/commits/178b5ffba316164fba86b86a67e9402b226fbc48

Reply:
Wait, why do we need deprecation functionality? We already have aliases (see behaviour vs behavior), and all the aliases but one are effectively deprecated -- they don't show up in the scheme. If you want to do anything else for deprecated aliases, I think you can build off that functionality?

Reply:
Right I guess we don't need that then? For some reason I thought aliases only worked for the last part of config names. I'll change that back then

Reply:
>For some reason I thought aliases only worked for the last part of config names.

They might, I dont' remember. But, even that's the case, we should extend that functionality, rather than add a new, similar one. Might be a good idea to document this!